datamanager/__init__.py

```diff
@@ -17,6 +17,7 @@ class DataManager(
 ):
     storage_provider: str = "local"
     volume_name: str = "/tmp"
+    volume_account: str = None
```

Comment:
Let's remove this field. We don't need it anymore.

Reply:
Removed and AWS is working fine without it